lib/common/options.js

```diff
@@ -69,11 +69,11 @@ Options.from = function (_options) {
     .map(dir => path.resolve(options.rootDir, dir))
 
   // Sanitize extensions
-  if (options.extensions.indexOf('js') === -1) {
+  if (!options.extensions.includes('js')) {
```

Comment:
```js
options.extensions = [
  ["js", "mjs"].filter(ext => !options.extensions.includes(ext)),
  ...options.extensions,
]
```
Or, if we performance-thot for no apparent reason:
```js
const mandatoryExtensions = ["js", "mjs"]
for (const ext in options.extensions) {
  const i = mandatoryExtensions.indexOf(ext)
  if (i !== -1) {
    if (mandatoryExtensions.length === 1) break // LMAO will this save us when the input already starts with ['js', 'mjs',... ? Or will we just deoptimize the function? No one knows LMAO
    mandatoryExtensions.splice(i, 1)
  }
}
options.extensions = mandatoryExtensions.concat(options.extensions)
```

Reply:
This is certainly a bottleneck! Lets optimize hell out of it.
kappa xD

Reply:
Have a better solution for it

Reply:
Better than filter -> concat?

Reply:
Your solution won't work though

```js
const invalid = ["js","mjs"]

const extensions = ["css","scss","ts","js"]

const res = invalid
  .filter(ext => !extensions.includes(ext))
  .concat(extensions)

console.log(res)

// [ 'mjs', 'css', 'scss', 'ts', 'js' ]

```

Reply:
You are right